physical_property/pKa/README.md

```diff
@@ -8,4 +8,4 @@ Detailed instructions and a submission template for the pK<sub>a</sub> challenge
 Experimental pK<sub>a</sub> measurements will made available after the challenge deadline.
 
 ## Manifest
-*Under construction*
+- [`microstates/`](microstates/) - This directory contains `.CSV` files that list microstate IDs and canonical isomeric SMILES of microstates. Files are separated by molecule ID. Updated microstates and their microstate IDs can be found in `SMXX_microstates.csv` files.
```

Comment:
Should you say something in these files about how people ought to submit additional microstates if they want to consider ones not enumerated here? 

(Do they need to submit them at this point? Not sure if we've thought this through. Possibly it would be adequate  to include them in their submission file. Have you thought through this? May want to consult with Mehtap cc me.)

Reply:
Will address in separate PR.